Tests fail when running on DB's with certain locales

http://www.postgresql.org/docs/9.1/static/locale.html notes that databases uses locale specific sorting. Because the tests do not create their own database, they are dependent on the locale of the database. This breaks the sorting test.

Using xxx_pattern_ops (http://www.postgresql.org/docs/9.1/interactive/indexes-opclass.html) the test could enforce English sorting.

```
simple-query-tests.js.....Message: undefined
AssertionError: "Brian" == "Aaron"
at /var/www/node-postgres/test/integration/client/simple-query-tests.js:33:14
at test (/var/www/node-postgres/test/test-helper.js:145:16)
at null.<anonymous> (/var/www/node-postgres/test/integration/client/simple-query-tests.js:32:5)
at assert.emits (/var/www/node-postgres/test/test-helper.js:46:16)
at g (events.js:185:14)
at EventEmitter.emit (events.js:88:17)
at p.handleReadyForQuery (/var/www/node-postgres/lib/query.js:81:8)
at p.connect.con.on.self.activeQuery (/var/www/node-postgres/lib/client.js:121:24)
at EventEmitter.emit (events.js:115:20)
at Socket.p.connect (/var/www/node-postgres/lib/connection.js:75:14)
```

--- a/test/integration/client/api-tests.js
+++ b/test/integration/client/api-tests.js
@@ -26,7 +26,7 @@ suite.test('callback API', done => {
   client.query(config)
   client.query('INSERT INTO peep(name) VALUES ($1)', ['aaron'])
 
-  client.query('SELECT * FROM peep ORDER BY name', (err, res) => {
+  client.query('SELECT * FROM peep ORDER BY name COLLATE "C"', (err, res) => {
     assert(!err)
     assert.equal(res.rowCount, 3)
     assert.deepEqual(res.rows, [

--- a/test/integration/client/prepared-statement-tests.js
+++ b/test/integration/client/prepared-statement-tests.js
@@ -122,7 +122,7 @@ var suite = new helper.Suite()
   suite.test('with small row count', function (done) {
     var query = client.query(new Query({
       name: 'get names',
-      text: 'SELECT name FROM zoom ORDER BY name',
+      text: 'SELECT name FROM zoom ORDER BY name COLLATE "C"',
       rows: 1
     }, done))
 
@@ -132,7 +132,7 @@ var suite = new helper.Suite()
   suite.test('with large row count', function (done) {
     var query = client.query(new Query({
       name: 'get names',
-      text: 'SELECT name FROM zoom ORDER BY name',
+      text: 'SELECT name FROM zoom ORDER BY name COLLATE "C"',
       rows: 1000
     }, done))
     checkForResults(query)

--- a/test/integration/client/simple-query-tests.js
+++ b/test/integration/client/simple-query-tests.js
@@ -6,7 +6,7 @@ var Query = helper.pg.Query
 test('simple query interface', function () {
   var client = helper.client()
 
-  var query = client.query(new Query('select name from person order by name'))
+  var query = client.query(new Query('select name from person order by name collate "C"'))
 
   client.on('drain', client.end.bind(client))
 
@@ -43,7 +43,7 @@ test('prepared statements do not mutate params', function () {
 
   var params = [1]
 
-  var query = client.query(new Query('select name from person where $1 = 1 order by name', params))
+  var query = client.query(new Query('select name from person where $1 = 1 order by name collate "C"', params))
 
   assert.deepEqual(params, [1])
 

--- a/test/native/evented-api-tests.js
+++ b/test/native/evented-api-tests.js
@@ -64,7 +64,7 @@ test('parameterized queries', function () {
 
   test('multiple parameters', function () {
     var client = setupClient()
-    var q = client.query(new Query('SELECT name FROM boom WHERE name = $1 or name = $2 ORDER BY name', ['Aaron', 'Brian']))
+    var q = client.query(new Query('SELECT name FROM boom WHERE name = $1 or name = $2 ORDER BY name COLLATE "C"', ['Aaron', 'Brian']))
     assert.emits(q, 'row', function (row) {
       assert.equal(row.name, 'Aaron')
       assert.emits(q, 'row', function (row) {

